tests/ecmp/test_fgnhg.py

```diff
@@ -233,6 +328,36 @@ def fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank
     for dst_ip in dst_ip_list:
         partial_ptf_runner(ptfhost, 'initial_hash_check', dst_ip, exp_flow_count)
 
+    if WARM_BOOT_TESTING == True:
+        warm_reboot_start_time = time.asctime( time.localtime(time.time()) ) 
+        logger.info ("warm reboot start time is ")
+        logger.info (warm_reboot_start_time)
+
+        #Async Thread for Warm Boot
+        pool = ThreadPool()
+        pool.apply_async(reboot, args = (duthost, localhost, "warm", ))
+
+        #Async Thread for Continuously Reprogramming ECMP Routes via vtysh
+        pool1 = ThreadPool()
+        pool1.apply_async(reprogram_ecmp_routes, args = (duthost, prefix_list, ip_to_port, ipcmd, ))
+
+        #Async Thread for Continuously Checking ECMP Routes under Warm Boot condition
+        pool2 = ThreadPool()  
```

Comment:
The comment here is a little bit misleading, recommend stating it as: "Thread to continuously send packets while warm reboot is ongoing, and validate datapath for Fine Grained ECMP"

Reply:
@kktheballer can you please mark the fixed comments as resolved?